src/auth/session/storage/custom.ts

```diff
@@ -70,11 +82,58 @@ export class CustomSessionStorage implements SessionStorage {
 
   public async deleteSession(id: string): Promise<boolean> {
     try {
-      return await this.deleteCallback(id);
+      return await this.deleteSessionCallback(id);
     } catch (error) {
       throw new ShopifyErrors.SessionStorageError(
         `CustomSessionStorage failed to delete a session. Error Details: ${error}`,
       );
     }
   }
+
+  public async deleteSessions(ids: string[]): Promise<boolean> {
+    if (this.deleteSessionsCallback) {
+      try {
+        return (await this.deleteSessionsCallback(ids)) as boolean;
+      } catch (error) {
+        throw new ShopifyErrors.SessionStorageError(
+          `CustomSessionStorage failed to delete array of sessions. Error Details: ${error}`,
+        );
+      }
+    } else {
+      throw new ShopifyErrors.SessionStorageError(
```

Comment:
I wonder if throwing an error makes sense here, because we're essentially making the optional functions mandatory if we blow up when they're called. Would it make sense to just log something here instead?